docs/internal/investigations/secure-resolvers/secure-resolvers-v2.md

```diff
@@ -1,39 +1,41 @@
 
-A runtime must be not be able to modify other runtime's metadata in the same tenant by default. A runtime must also not be able to fetch applications that belong to other runtimes.
+By default, a runtime must not be allowed to modify another runtime's metadata within the same tenant. Additionally, a runtime must not be allowed to fetch applications that belong to other runtimes.
 
-An integration system must be not be able to modify other integration system's metadata by default.
+By default, an integration system must not be allowed to modify another integration system's metadata.
 
-Integration systems in general can manage other systems and their metadata (applications and runtimes). An integration system should be able to modify only what it has created or what it was granted access to modify. For example, if an integration system registers an application, it can later also register packages for this application. However, it cannot modify metadata for applications that other consumers registered unless explicitly being granted access to do so. Another example would be when a user registers an application via the UI. In order for integration system X to get access to this application either the user must grant this access via an API call to Director (grant access to the integration system for this application) or some automated procedures must be in place (there is a concept about stacking credentials via one time tokens in the later sections of the document). While the examples so far are with applications, these requirements apply also for application subentities, application templates and runtimes managed by integration systems.
+In general, integration systems can manage other systems and their metadata (such as applications, and runtimes). However, an integration system must be allowed to modify only entities that it has created, or entities to which it has granted access for modifications. For example, if an integration system registers an application, later on, it can  also register packages for this application. However, it cannot modify metadata for applications that other consumers registered, unless it has granted access explicitly to do so. Another example is when a user registers an application via the UI. To get access to this application, the integration system X either must be granted this access by the user via an API call to Director (grant access to the integration system for this application), or some automated procedures must be in place (a concept about stacking credentials via one time tokens is available in the further sections of this document). Although the given examples are related to applications, these requirements are also valid for application sub-entities, application templates, and runtimes managed by integration systems.
```

Comment:
or entities to which it has granted access for modifications -> or entities to which it has *been* granted access for modification

Reply:
unless it has granted access explicitly to do so ->  unless it has granted *been* access explicitly to do so